frontend/src/App.tsx

```diff
@@ -14,6 +14,7 @@ export default function App() {
           <Routes>
             <Route path="/" element={<Home />} />
             <Route path="/login" element={<Login />} />
+            <Route path="/unit/{id}" element={<UnitDetails />} />
```

Comment:
```suggestion
            <Route path="/unit/:id" element={<UnitDetails />} />
```

See [React Router Dynamic Segments docs](https://reactrouter.com/en/main/route/route#dynamic-segments)